.github/workflows/release.yml

```diff
@@ -77,7 +77,7 @@ jobs:
           go get -u github.com/sanbornm/go-selfupdate/...
 
       - name: Install Taskfile
-        uses: arduino/actions/setup-taskfile@master
+        uses: arduino/setup-task@v1
         with:
           version: '3.x'
```

Comment:
```suggestion
```
maybe it's better to remove it since it's the default value

Reply:
It's currently the default, but at some point after Task 4.0.0 comes out, that will become the default.

The benefit to specifying the 3.x version is that a major version bump of Task indicates there has been a breaking change, which might also break the workflows.

The benefit to not specifying the version is it avoids either needing to make the effort to keep the workflows updated to benefit from the modern Task, or else allowing them to use an outdated Task version.

So there are advantages and disadvantages either way. I will let you decide which is the better approach. If you want to use the default, I'll be happy to update this PR.

Reply:
I did not thought about the future development.
You are right… Having it pinned it's a better approach :+1: